inngest/src/main/kotlin/com/inngest/ktor/Route.kt

```diff
@@ -62,21 +62,15 @@ fun Route.serve(
                 val body = call.receiveText()
                 try {
                     val response = comm.callFunction(fnId, body)
-                    call.response.header(
-                        HttpHeaders.ContentType,
-                        ContentType.Application.Json.toString(),
-                    )
+                    response.headers.forEach({ (k, v) -> call.response.header(k, v) })
                     call.response.status(
                         HttpStatusCode(response.statusCode.code, response.statusCode.message),
                     )
-//                    println("response: " + response.body)
-                    call.respond(response.body)
+                    call.respondText(response.body)
```

Comment:
interesting, so technically what was here before, `call.respond(response.body)` would have called the serializer for the type of `response.body` which is just a string? I think this change signals intent better

Reply:
exactly yeah, `respond` is typically used with serializable objects.